lock-api/src/main/java/com/palantir/lock/client/NamespacedConjureTimelockService.java

```diff
@@ -34,8 +38,12 @@
 public interface NamespacedConjureTimelockService {
     ConjureUnlockResponse unlock(ConjureUnlockRequest request);
```

Comment:
Is there a reason to keep this API? I get that breaking APIs is a "bad thing"™️, but we actively don't want usages after this change

Reply:
I think the intention was to make the smallest possible change given tight timelines, though happy to do this in a follow up